examples/liveness/README.md

```diff
@@ -0,0 +1,27 @@
+This example shows two types of pod health checks: HTTP checks and containter execution checks. This [guide](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/walkthrough/k8s201.md) has more information on health checks.
```

Comment:
You can link directly to the health checking section: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/walkthrough/k8s201.md#health-checking

You could also summarize what the example does before going into the details. Maybe a sentence or two explaining what  the command is doing `echo ok > /tmp/health; sleep 10; rm -rf /tmp/health; sleep 600`.